dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -234,3 +234,7 @@ scheme.incorrect.error.useraction=The scheme type must be http, https, ws, or ws
 upgradeHttpToWebSocket.failed=CWWKH0055E: The jakarta.websocket.server.ServerContainer#upgradeHttpToWebSocket call failed due to {0}. 
 upgradeHttpToWebSocket.failed.explanation=The server could not upgrade the HTTP or HTTPS request to the ws/wss protocol.
 upgradeHttpToWebSocket.failed.useraction=Review the error to identify why the request failed to upgrade.
+
+maxmessagesize.exceeded=CWWKH0056E: The maxMessageSize for method {0} exceeded Integer.MAX_VALUE for class {1}.
+maxmessagesize.exceeded.explanation=The OnMessage annotation parameter maxMessageSize can only be set to a maxmimum value of Integer.MAX_VALUE as per the WebSocket 2.2 specification.
+maxmessagesize.exceeded.useraction=Reduce maxMessageSize to be a value less than Integer.MAX_VALUE.
```

Comment:
Suggested change:

Reduce the maxMessageSize parameter value to no more than Integer.MAX_VALUE.

(maxMessageSize can be equal to Integer.MAX_VALUE and not just less than.)